prow/config.yaml

```diff
@@ -19,35 +19,35 @@ presubmits:
     always_run: true
     context: pull-cadvisor-e2e
     rerun_command: "@k8s-bot pull-cadvisor-e2e test this"
```

Comment:
You should change the suggested `rerun_command` to `/test pull-cadvisor-e2e`.

Also, please write unit tests that validate the following continue to work:
Already exist:
`@k8s-bot JOB test this`
`@k8s-got test this`  
Needed
`/test JOB`
`/test`